.github/workflows/kakarot_rpc.yml

```diff
@@ -109,3 +109,67 @@ jobs:
       - name: Inspect image
         run: |
           docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:latest
+
+  build-hive:
```

Comment:
does this work when you push to your local main branch?

Reply:
Yup image successfully built [here](https://github.com/alamatxyz/kakarot-rpc/pkgs/container/kakarot-rpc%2Fnode-hive)